### DISCUSSION TITLE
discord py 1.5

### DISCUSSION THREAD

**Discussion by tangsan1337** [Rapptz/discord.py#5869] [Q&A] [answered]
How can i get all users from my server?
if i user code:
for i in guild.members:
    yield i
displays only bot.

In version 1.4.1 i was getting all users from my server, how can i fix it?

**Comment by Rapptz** [comment, marked as answer]
This is documented here: https://discordpy.readthedocs.io/en/latest/intents.html#where-d-my-members-go

**Reply by tangsan1337** [reply]
thank you
